server/app/controllers/api/v1/ckb_transactions_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class CkbTransactionsController < ApplicationController
+      def show
+        ckb_transaction = CkbTransaction.find_ckb_transaction(params[:id])
```

Comment:
Why not use `find` or `find_tx`, `CkbTransaction` and `ckb_transaction` has a little bit of repetition.

Reply:
because I will check the format of tx hash.

Reply:
I means rename this method

Reply:
ok, I got it.